sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -45,11 +45,11 @@ statement
     | ALTER DATABASE identifier SET DBPROPERTIES tablePropertyList     #setDatabaseProperties
     | DROP DATABASE (IF EXISTS)? identifier (RESTRICT | CASCADE)?      #dropDatabase
     | createTableHeader ('(' colTypeList ')')? tableProvider
-        (OPTIONS tablePropertyList)?
+        (OPTIONS optionParameterList)?
         (PARTITIONED BY partitionColumnNames=identifierList)?
         bucketSpec?                                                    #createTableUsing
     | createTableHeader tableProvider
-        (OPTIONS tablePropertyList)?
```

Comment:
I hope I understood this correctly. I guess you meant the `tableProperty` rule, for example,  as below:

```antlr
tableProperty
    : key=tablePropertyKey (EQ? value= optionValue)?
    ;
```

If so, I am worried if this affects other rules such as `DBPROPERTIES` and `TBLPROPERTIES` (allowing other types as values). I made this separate because it seems allowing other types in `OPTIONS` clause complies standard SQL.

If not, could you give me an advice in a bit more detail please?